src/Catalyst.Node.POA.CE.Tests/UnitTests/Config/ConfigCopierTests.cs

```diff
@@ -0,0 +1,144 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Catalyst.Core.Lib.Config;
+using Catalyst.Protocol.Network;
+using Catalyst.TestUtils;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Catalyst.Node.POA.CE.Tests.UnitTests.Config
+{
+    public sealed class ConfigCopierTests : FileSystemBasedTest
+    {
+        public ConfigCopierTests(ITestOutputHelper output) : base(output) { }
+
+        private sealed class ConfigFilesOverwriteTestData : TheoryData<string, NetworkType>
+        {
+            public ConfigFilesOverwriteTestData()
+            {
+                Add(Constants.NetworkConfigFile(NetworkType.Mainnet), NetworkType.Mainnet);
+                Add(Constants.NetworkConfigFile(NetworkType.Testnet), NetworkType.Testnet);
+                Add(Constants.NetworkConfigFile(NetworkType.Devnet), NetworkType.Devnet);
+                Add(Constants.SerilogJsonConfigFile, NetworkType.Devnet);
+            }
+        }
+
+        [Theory]
+        [ClassData(typeof(ConfigFilesOverwriteTestData))]
+        [Trait(Traits.TestType, Traits.IntegrationTest)]
+        public void RunConfigStartUp_Should_Not_Overwrite_An_Existing_Config_File(string moduleFileName,
+            NetworkType network)
+        {
+            RunConfigStartUp_Should_Not_Overwrite_Existing_Files(moduleFileName, network);
+        }
+
+        private void RunConfigStartUp_Should_Not_Overwrite_Existing_Files(string fileName,
+            NetworkType network = NetworkType.Devnet)
+        {
+            var currentDirectory = FileSystem.GetCatalystDataDir();
+            currentDirectory.Create();
+            currentDirectory.Refresh();
+            var existingFileInfo = new FileInfo(Path.Combine(currentDirectory.FullName, fileName));
+            if (existingFileInfo.Directory != null && !existingFileInfo.Directory.Exists)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)